public/app/plugins/datasource/cloudwatch/tracking.ts

```diff
@@ -124,5 +139,14 @@ export const onDashboardLoadedHandler = ({
   }
 };
 
+type SampleQueryTrackingEvent = {
+  queryLanguage: LogsQueryLanguage;
+  queryCategory: string;
+};
+
+export const trackSampleQuerySelection = (props: SampleQueryTrackingEvent) => {
+  reportInteraction('cloudwatch-logs-cheat-sheet-query-clicked', props);
```

Comment:
```suggestion
  const { queryLanguage, queryCategory } = props;
  reportInteraction('cloudwatch-logs-cheat-sheet-query-clicked', { queryLanguage, queryCategory });
```

minor nit: for tracking, it would be good to establish a pattern of explicitly building up what we intend to track. That way we prevent any extra properties from being sent in the payload.